doc/manuals/administration_and_configuration/batch/annex_a_create_and_install_keypair.md

```diff
@@ -8,4 +8,4 @@ You will be prompted for a file path containing the private key and a passphrase
 
 In order to install this key pair for remote authenticated accesses into a certain host, just copy the public key in the ssh authorized keys file owned by the user we want to be authenticated in the host:
 
-    $ cat `<key_name>`.pub >> /home/`<user>`/.ssh/authorized_keys
+    $ cat <key_name>.pub >> /home/<user>/.ssh/authorized_keys
```

Comment:
Probably this change makes sense to be donde in the overall documentation, in a single shoot. Maybe you could create a tech debt issue about it (up to you).

NTC